CommonMark/Parser/BlockMethods.cs

```diff
@@ -295,6 +298,9 @@ public static void ProcessInlines(Block block, Dictionary<string, Reference> ref
                         sc.FillSubject(subj);
                         delta = subj.Position;
 
+                        var parsers = parseEmphasisInIndentedCode
+                            ? settings.InlineEmphasisParsers
+                            : settings.InlineParsers;
```

Comment:
I would prefer the `settings` property accesses to be brought out of the cycles to the top of the method. Though most probably the performance implication is very, very tiny...